### DISCUSSION TITLE
Unable to access 60 day free trial

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33803] [Copilot Conversations]
Hey all. I'm trying to access the 60 day free trial, I have not previously had a trial. When I attempt to do so I am informed I will be immediately charged the monthly subscription price. Please see [this video](https://nimb.ws/OkpDhZ). Does anyone have any idea? Thanks!

**Comment by AyushShahh** [comment]
Before starting a paid subscription, you can set up a one-time 60-day trial to evaluate GitHub Copilot. To begin a trial, you will need to choose a monthly or yearly billing cycle, and provide a payment method. If you do not cancel the trial before the end of the 60 days, the trial will automatically convert to a paid subscription. You can cancel your GitHub Copilot trial at any time during the 60 days and you won't be charged. If you cancel before the end of the trial, you will continue to have access to GitHub Copilot until the 60-day trial period ends.

**Reply by AyushShahh** [reply]
https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot

**Comment by ghost** [comment]
Thanks for the additional clarification @AyushShahh. However, I do not understand why I am seeing this on the signup page if I am not charged for 60 days.
<img width="785" alt="Screenshot 2022-09-22 123602" src="https://user-images.githubusercontent.com/13105021/191736615-57940bef-2d77-4eda-a56f-4e904d9747f6.png">

**Reply by AyushShahh** [reply]
This [article](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription) might be useful for you

**Reply by ghost** [reply]
Thank you. It is time for me to hang my head in shame! I've just discovered that the reason I cannot access the trial is that I already had one and completely forgot! So, sincere apologies for wasting your time!

**Reply by AyushShahh** [reply]
No worries
